src/resources/base.js

```diff
@@ -0,0 +1,4 @@
+/** @provideGoog */
```

Comment:
I don't actually see any special handling for the provideGoog annotation in the compiler. As far as I can tell that annotation is understood by some other (Google-internal) tools but not by the compiler itself.